docs/content/en/integrations/parsers/file/crunch42.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Crunch42 Scan"
+toc_hide: true
+---
+Import JSON findings from Crunch42 vulnerability scan tool.
```

Comment:
`Import JSON findings from [Crunch42](https://42crunch.com) vulnerability scan tool.`

Reply:
Maybe you can get a little more specific regarding which output is provided as there are multiple offers from Crunch42